src/index.ts

```diff
@@ -319,6 +319,60 @@ export default class OneSignal {
       );
     }
   }
+   /* L I V E   A C T I V I T Y  */
+
+    /**
+   * Associates a temporary push token with an Activity ID on the OneSignal server.
+   * @param  {string} activityId
+   * @param  {string} token
+   * @param  {Function} handler
+   * @returns void
+   */
+  static enterLiveActivity(
+    activityId: string,
+    token: string,
+    handler?: Function,
+  ): void {
+    if (!isNativeModuleLoaded(RNOneSignal)) {
+      return;
+    }
+
+    // Android workaround for the current issue of callback fired more than once
+    if (!handler && Platform.OS === 'ios') {
```

Comment:
Shouldn't we just check for `!handler` here?, we already have a `Platform.OS === 'ios'` check below.